slik_wrangler_app/pages/streamlit.py

```diff
@@ -275,6 +277,14 @@ def main():
                     with st_stdout("info"):
                         transformed_df = pp.featurize_datetime(transformed_df,choice,feat_choice, drop)
 
+            if get_attr_fn:
+                #choice = st.sidebar.selectbox('select target column', transformed_df.columns.tolist())
+                col_list_ap = col_list.append(None)
+                index = len(col_list) -1
+                choice = st.sidebar.selectbox('exclude target column', col_list,key='get_attr',help="""
```

Comment:
@Justus-coded why exclude target column. Can be select target column instead. cc @AdesholaAfolabi

Reply:
Wanted to use something more descriptive. "Select target column" doesn't let the user know that the target column will be excluded from the features.

Reply:
Yeah I saw. Can you revert to the previous return statement and handle the appearance on the streamlit script. You can return both objects and store in a dictionary instead

Reply:
Alright

Reply:
@Sensei-akin @Justus-coded I see the PR has been merged. Were the changes fixed before merging the PR?